packages/integration-tests/src/fixtures/cluster/default-config.ts

```diff
@@ -1,10 +1,29 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 
 const defaultConfig = {
-  cluster_network_cidr: '10.128.0.0/14',
-  cluster_network_host_prefix: 23,
-  inactive_deletion_hours: 480,
-  ntp_source: '',
-  service_network_cidr: '172.30.0.0/16',
+    cluster_network_cidr: '10.128.0.0/14',
```

Comment:
Bringing changes from API to  `defaultConfig` to avoid the UI thinking that the user is using advanced configuration settings.